loadtest/control/simplecontroller/actions.go

```diff
@@ -100,6 +100,55 @@ func (c *SimpleController) createPost() control.UserStatus {
 	return c.newInfoStatus(fmt.Sprintf("post created, id %v", postId))
 }
 
+func (c *SimpleController) addReaction() control.UserStatus {
+	// get posts from UserStore that have been created in the last minute
+	posts, err := c.user.Store().PostsSince(time.Now().Unix()*1000 + 60000)
+	if err != nil {
+		return c.newErrorStatus(err)
+	}
+	if len(posts) == 0 {
+		return c.newInfoStatus("no posts to add reaction to")
+	}
+
+	err = c.user.SaveReaction(&model.Reaction{
+		UserId:    c.user.Store().Id(),
+		PostId:    posts[0].Id,
+		EmojiName: "grinning",
+	})
+
+	if err != nil {
+		return c.newErrorStatus(err)
+	}
+
+	return c.newInfoStatus("added reaction")
+}
+
+func (c *SimpleController) removeReaction() control.UserStatus {
+	// get posts from UserStore that have been created in the last minute
+	posts, err := c.user.Store().PostsSince(time.Now().Unix()*1000 + 60000)
+	if err != nil {
+		return c.newErrorStatus(err)
+	}
+	if len(posts) == 0 {
+		return c.newInfoStatus("no posts to remove reaction from")
+	}
+
+	reactions, err := c.user.Store().Reactions(posts[0].Id)
```

Comment:
Maybe we can iterate through it to check for reactions because we can have 10 posts in that minute and the 5th one is the one with reactions. What do you think?

Reply:
We could but I also think it's fine to return if nothing is found. I am trying not to put too much logic into what the `SimpleController` does at the moment. We can always go back to this later to make it more realistic.